### PR TITLE
fix(MultiSig)!: change interface and substitute adaptive_address for autonomy

### PR DESCRIPTION
**Pull request by LycrusHamster**
**What type of PR is this?**

fix(multi-sig)!: change interface and substitute adaptive_address for autonomy

BREAKING CHANGE: the interface is changed and affect on all sdk/clients

**What this PR does / why we need it**:

due to the bech32 format, the adaptive address is not suitable.

so we add 'autonomy' field.

B.T.W. change the UpdateAccountPayload to make it non-ambigious
B.T.W. change the genesis payload to remove ambiguity.

Docs is here
https://github.com/HuobiGroup/huobi-chain-docs/pull/8

Java sdk:
https://github.com/nervosnetwork/muta-sdk-java/pull/24

> due to 2 different iteration, the pr of java sdk has been merged. This is a exclusion this time

Js sdk:

https://github.com/nervosnetwork/muta-sdk-js/pull/23

### CODE REVIEW

File: built-in-services/multi-signature/src/lib.rs [hunk 1]
```diff
@@ -119,11 +114,7 @@ impl<SDK: ServiceSDK> MultiSignatureService<SDK> {
             })
             .collect::<Vec<_>>();
 
-        let owner = if payload.owner == ADEPTIVE_ADDRESS.clone() {
-            address.clone()
-        } else {
-            payload.owner.clone()
-        };
+        let owner = payload.owner;
```
Comment (KaoImin) [inline built-in-services/multi-signature/src/lib.rs:117]:
```suggestion
```
redundancy binding

**Comment by KaoImin** [comment]
I think the type of this PR is `refactor` rather than `fix`.

**Comment by yejiayu** [comment]
Will this PR have BRACKING_CHANGE? Why does not add `!` to the title? Does the toolchain need to adapt?

@LycrusHamster @homura

**Comment by LycrusHamster** [comment]
> Will this PR have BRACKING_CHANGE?
> Why does not add `!` to the title?
> Does the toolchain need to adapt?
> 
> @LycrusHamster @homura

Yes, now question mark has been added and BREAKING CHANGE have been explicitly add in the commit msg

**Comment by yejiayu** [comment]
/hold

Wait SDK adaption

**Pull request merged**